conf/junit/node-default.properties

```diff
@@ -5,4 +5,6 @@
 # Mock mining for junit tests, in-memory db (offline and accepting any nonce as valid)
 node.network = signum.net.MockNetwork
 
-DB.Url=jdbc:h2:mem:signum-mock;DB_CLOSE_ON_EXIT=FALSE
+#DB.Url=jdbc:h2:mem:signum-mock;DB_CLOSE_ON_EXIT=FALSE
```

Comment:
well, using h2 memdb for mock mining/unit tests is not a bad idea...If you use sqlite then it should be wiped on each test run... is this given? Or we start supporting sqlite mem db also.